src/renderer/pages/Settings/Networks/components/NetworkSelector/NetworkSelector.tsx

```diff
@@ -38,10 +39,14 @@ export const NetworkSelector = ({
   onDisconnect,
   onConnect,
   onRemoveCustomNode,
-  onChangeCustomNode,
 }: Props) => {
   const { t } = useI18n();
   const [ref, scroll] = useScrollTo<HTMLDivElement>(TRANSITION_DURATION);
+  const openAddCustomNodeModal = useUnit(addCustomRpcModel.events.processStarted);
```

Comment:
Not needed to use `useUnit` for events, just call `addCustomRpcModel.events.processStarted()`